cmd/server.go

```diff
@@ -151,8 +203,26 @@ func initServerCmd() {
 		Args:    cobra.ExactArgs(1),
 		Run:     produceServerCmdRunFunc(client, serverDeleteAction),
 	}
+	var createCmd = &cobra.Command{
+		Use:     "create [flags]",
+		Example: `./gscloud server create --name "Server Name" --cpu 6 --mem 4 --with-template "Debian 10" --password "p4ssw0rd" --hostname "gridscale"`,
+		Aliases: []string{"create"},
```

Comment:
No need to give an alias here.

Reply:
##### Good catch. Thanks, I somehow didn't see this.
- Removed all unnecessary aliases i found.